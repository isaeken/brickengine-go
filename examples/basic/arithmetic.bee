let a = 5 + 10 * 2;
let b = (5 + 10) * 2;

let output = {
  result_a: a,
  result_b: b
}

return output
