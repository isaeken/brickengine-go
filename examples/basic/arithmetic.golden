map[result_a:30 result_b:30]
